node/src/cli.rs

```diff
@@ -143,9 +143,13 @@ pub fn load_spec(
             #[cfg(feature = "parachain")]
             para_id,
         )?),
-        "battery_park" => Box::new(crate::chain_spec::battery_park_config(
-            #[cfg(feature = "parachain")]
-            para_id,
```

Comment:
Rust-based specifications are useful for development and small tweaks although most (all?) parameters can also be modified directly in the `json` variant. For deployment, it would just be a matter of passing the canonical spec file `--chain path/to/file.json`.
Polkadot uses the same strategy for all runtimes. For example, Rococo -> https://github.com/paritytech/polkadot/blob/f9d5ad2a771319ec59265f434789b8080a442c9c/node/service/src/chain_spec.rs#L1000

In any case, this method is fine and will probably be sufficient